rbac/management/principal/view.py

```diff
@@ -183,12 +182,13 @@ def get(self, request):
                 count = len(data)
             else:
                 count = None
+            last_link = int(count) - int(limit) if (int(count) - int(limit)) >= 0 else 0
             response_data["meta"] = {"count": count, "limit": limit, "offset": offset}
             response_data["links"] = {
                 "first": f"{path}?limit={limit}&offset=0{usernames_filter}",
                 "next": f"{path}?limit={limit}&offset={offset + limit}{usernames_filter}",
                 "previous": f"{path}?limit={limit}&offset={previous_offset}{usernames_filter}",
-                "last": None,
+                "last": f"{path}?limit={limit}&offset={last_link}{usernames_filter}",
```

Comment:
it is needed to fix all links here to work in same way as for other endpoints like `GET /groups/` for example `GET /principals/` for limit=10 and offset=0 (default values) the previous link should be `null`